.github/workflows/ci.yml

```diff
@@ -26,13 +26,16 @@ jobs:
           coverage: none
 
       - name: Install PHP 7 dependencies
-        run: composer update --prefer-dist --no-interaction --no-progress
+        run: |
+          composer update --prefer-dist --no-interaction --no-progress
+          vendor/bin/simple-phpunit install
```

Comment:
The repo contains a few unit tests but they're never run by the CI, isn't it?
I'm adding simple-phpunit to run them - at least run the ones that don't require one of the well-know deps. This allows me to test the composer plugin using phpunit (I don't know phpspec, dunno if it's possible to use it for that - it'd need help if phpspec is a requirement.)

Reply:
i fear you are correct that we missed running the phpunit tests. phpspec is no requirement - the original authors started with phpspec, but we do have some phpunit tests and i myself am more familiar with that tool and don't know phpspec well.